CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+This is a simple guide of the workflow being used in Cryostat. We will use https://github.com/cryostatio/cryostatio.github.io as an example.
+
+## Fork the upstream repository
+
+See https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+## Local changes
+
+Clone the forked repository.
+```bash
+$ git clone git@github.com:<username>/cryostatio.github.io.git
+$ git remote add upstream git@github.com:cryostatio/cryostatio.github.io.git # Adding upstream as remote
+```
+
+Checkout another branch.
```

Comment:
is this checking out a new branch? suggest "Navigate to Cryostat directory and checkout new branch" if so